### PR TITLE
Simplify cross-compilation for scalafix output

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
 Several levels of contribution are possible!
 
 ### Report a missing case
 
-Create an issue tagged with the
-[migration](https://github.com/scala/collection-strawman/labels/migration) label.
+Create an issue [scala-collection-compat/issues](https://github.com/scala/scala-collection-compat/issues).
 Embrace `diff`s to describe differences between the standard collections and
-the strawman:
+the new colloction:
```
Comment (julienrf) [inline CONTRIBUTING.md]: typo: coll**o**ction

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -63,55 +63,35 @@ Then maybe someone will take over and implement your use case… or maybe you wi
...
-│           └── scala       | Rule implementations
-└── tests                   |
-    └── src                 |
-        └── test            |
-            └── scala       | Scalafix testkit launcher (useful to run a single input file)
++------------------+-----+-----------------+
+|                  |Input|     Output      |
+|                  | 2.12| 2.11| 2.12| 2.13|
++------------------+-----+-----+-----+-----+
+|data              |  X  |  X* |  X  |  X  |
```
Comment (julienrf) [inline CONTRIBUTING.md]: What does the star mean?

Comment (MasseGuillaume) [reply CONTRIBUTING.md]: The only exception is in this next PR: https://github.com/scala/scala-collection-compat/pull/122/files#diff-305e21c374e13a65d851eaf004df6a39

**Comment by MasseGuillaume** [comment]
CI timed out in Scala.js, unrelated to this PR.

**Pull request merged**